File: 4th Semester/Systems for Design and Implementation/C#/Homework1/OnwardTravel/models/ReservedSeat.cs
﻿namespace OnwardTravel.models
{
    internal class ReservedSeat : Entity<Guid>
    {
        public int SeatNumber { get; }

        public ReservedSeat(Guid guid, int seatNumber)
            : base(guid)
        {
            SeatNumber = seatNumber;
        }

        public ReservedSeat(int seatNumber)
        {
            SeatNumber = seatNumber;
        }
    }
}

File: 4th Semester/Systems for Design and Implementation/Java/Homework1/src/main/java/OnwardTravel/models/ReservedSeat.java
package OnwardTravel.models;

import java.util.UUID;

public class ReservedSeat extends Entity<UUID> {
    private final int seatNumber;

    public ReservedSeat(UUID uuid, int seatNumber) {
        super(uuid);
        this.seatNumber = seatNumber;
    }

    public int getSeatNumber() {
        return seatNumber;
    }
}
